cli/cfncluster/cli.py

```diff
@@ -156,6 +159,29 @@ def main():
     pversion = subparsers.add_parser('version', help='display version of cfncluster')
     pversion.set_defaults(func=version)
 
+    # Add ssh and scp
+    pssh = subparsers.add_parser('ssh', description='run ssh command with username and ip address pre-filled. '
+                                                    'This command may be customized in the aliases section of the config file.'
+                                                    'You may use: {CFN_USER} {MASTER_IP} variables in the command.')
+    pssh.add_argument("cluster_name", type=str, default=None,
+                        help='name of the cluster to set variables for.')
+    pssh.add_argument("--dryrun", "-d", action='store_true', dest="dryrun", default=False,
+                         help='print command and exit.')
+    pssh.add_argument("--args", type=str, dest="args", default="",
```

Comment:
`*nargs` instead?, or can you not use that on a subparser?

Reply:
I'm now passing through all args using `parse_known_args()` so it doesn't throw an exception for unknown arguments.